core/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcherArguments.scala

```diff
@@ -95,6 +100,9 @@ private[mesos] class MesosClusterDispatcherArguments(args: Array[String], conf:
         "  -h HOST, --host HOST    Hostname to listen on\n" +
         "  -p PORT, --port PORT    Port to listen on (default: 7077)\n" +
         "  --webui-port WEBUI_PORT WebUI Port to listen on (default: 8081)\n" +
+        "  --webui-url  WEBUI_URL  The WebUI URL to be set in the underlying\n" +
```

Comment:
There are a trillion ways to set configuration in Spark, half of them are deprecated, and some only work with certain entrypoint classes.

Some entrypoints, like SparkSubmit, support setting arbitrary properties in commands w/ something like --conf.  MesosClusterDispatcher does not.  So I say we just rely on `--properties-file`, rather than adding a command argument just for this one property.  It's already arbitrary which properties are exposed as special command parameters, so let's not add a new one.